packages/web/src/Components/Chip.tsx

```diff
@@ -10,6 +10,8 @@ export interface IProps {
   component?: any;
   deleteIcon?: any;
   disabled?: boolean;
+  color?: any;
```

Comment:
This interface is gonna be consumed later. Refactor the properties to not use `any`, but the correct type.